libs/core/src/lib/timeline/components/timeline-node-body/timeline-node-body.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'fd-timeline-node-body',
+  templateUrl: './timeline-node-body.component.html',
```

Comment:
Can we add encapsulation and changeDetection?

Reply:
@N1XUS @Betrozov I added onPush changeDetection strategy, but do we need encapsulation here? We don't export any global style, these child components even doesn't have own styles?